packages/components/loading-spinner/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@commercetools-uikit/loading-spinner",
-  "description": "A spinner animation to indicate loading state (e.g pending requests) for the user.",
+  "description": "A spinner animation to indicate loading state (e.g pending requests) for the user. By default, there is a 1 second delay between when the component mounts and when it renders.",
```

Comment:
Hmm I don't think the "1s delay" info needs to be mentioned here. To provide additional information in the readme there is an "additional description" MD file that can be used for that.